app/models.py

```diff
@@ -73,6 +73,44 @@ class Event(BaseModel):
     updated_at: datetime = Field(default_factory=datetime.utcnow)
 
 
+class MetricPayload(BaseModel):
+    # metric object received from frontend
+    name: str
+    marksScored: int
+    maxQuestionsAllowedToAttempt: int
+    numAnswered: int
+    correctlyAnswered: int
+    wronglyAnswered: int
+    partiallyAnswered: int
+    attemptRate: float
+    accuracyRate: float
+
+
+class QuestionSetMetric(BaseModel):
+    name: str
+    marks_scored: int
+    max_questions_allowed_to_attempt: int
+    num_answered: int
+    num_skipped: int
+    num_correct: int
+    num_wrong: int
+    num_partially_correct: int
+    attempt_rate: float
+    accuracy_rate: float
+
+
+class SessionMetrics(BaseModel):
+    total_sets: int
```

Comment:
what is sets here? question sets? what does this metric represent about a session?

Reply:
no need to remove it